src/Core/Checkout/Cart/Order/OrderConverter.php

```diff
@@ -272,7 +272,6 @@ public function assembleSalesChannelContext(OrderEntity $order, Context $context
                 ->addAssociation('addresses');
 
             $customer = $this->customerRepository->search($customerCriteria, $context)->getEntities()->first();
-            \assert($customer?->getAddresses() !== null);
```

Comment:
So you fix the assertion, but removed the line? Why?

Reply:
Your code did not fail when the association was not present, and there was no PHPstan error. Does it seem to have no benefit?

Reply:
Like telgi, I assumed that assertions were enabled for tests - it should assert that the given customerId must be found if given :thinking:

Reply:
But there should be no asserts in production code where you really except that they should fail. They should be only used, to go around a typing issue with phpstan